champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Vengeance.java

```diff
@@ -0,0 +1,84 @@
+package me.mykindos.betterpvp.champions.champions.skills.skills.knight.passives;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.champions.Champions;
+import me.mykindos.betterpvp.champions.champions.ChampionsManager;
+import me.mykindos.betterpvp.champions.champions.skills.Skill;
+import me.mykindos.betterpvp.champions.champions.skills.types.PassiveSkill;
+import me.mykindos.betterpvp.core.combat.events.CustomDamageEvent;
+import me.mykindos.betterpvp.core.components.champions.Role;
+import me.mykindos.betterpvp.core.components.champions.SkillType;
+import me.mykindos.betterpvp.core.listener.BPvPListener;
+import org.bukkit.Sound;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
+
+@Singleton
+@BPvPListener
+public class Vengeance extends Skill implements PassiveSkill, Listener {
+
+    private int numHits;
```

Comment:
Num hits is never retrieved. Needs to override loadSkillConfig: `@Override
    public void loadSkillConfig(){
        numHits= getConfig("numHits", 3, Integer.class);
    }`

Reply:
Does not apply if this is not the maximum hits this skill applies for.